tests/foreman/ui/test_host.py

```diff
@@ -360,6 +361,8 @@ def test_positive_provision_end_to_end(self):
                 session,
                 name=hostname,
                 org=self.org_name,
+                loc=self.loc_name,
+                force_context=True,
```

Comment:
Why forcing context? afaik context will be changed if it's not matching, forcing it will basically click on org/loc names when they're selected just in case. Or what's the issue here?

Reply:
because without force context the location is not selected

Reply:
That's sad. Hope #5645 will fix that. It would be better to fix the issue globally instead of here, inside specific test. But #5645 at this point is pretty risky change which requires a lot of testing, so LGTM for now.